clients/python/agentic-sandbox-client/k8s_agent_sandbox/gke_extensions/snapshots/utils.py

```
@@ -297,9 +297,9 @@ def wait_for_pod_termination(
 
 
 def wait_for_pod_ready(
-    k8s_helper,
+    k8s_helper: Any,
     namespace: str,
-    get_pod_name_func,
+    get_pod_name_func: Callable[[], str | None],
```

Ruff (0.15.20)

[error] Undefined name `Callable`

(F821)